lib/cc/engine/analyzable_files.rb

```diff
@@ -1,39 +1,29 @@
 module CC
   module Engine
     class AnalyzableFiles
-      def initialize(directory, config)
-        @directory = directory
+      def initialize(config)
         @config = config
       end
 
       def all
         @results ||= if @config["include_paths"]
-          build_files_with_inclusions(@config["include_paths"])
+          include_paths = @config["include_paths"]
+          if include_paths == ["./"]
+            ["."]
+          else
+            filter_files(@config["include_paths"])
```

Comment:
Might as well re-use the `include_paths` local here instead of accessing `@config` again.